chat-server/.eslintrc.js

```diff
@@ -1,16 +1,18 @@
 module.exports = {
   env: {
     node: true,
+    "jest/globals": true,
```

Comment:
Does this add jest globals to any file, even beyond tests? Can we make an eslintrc file specific to tests? https://eslint.org/docs/latest/use/configure/configuration-files#cascading-and-hierarchy

Reply:
yea we can scope the jest globals to test files. 

though, i think using `overrides` with in the same eslintrc file would be the most ESLint-idiomatic way to do it - https://eslint.org/docs/latest/use/configure/configuration-files#how-do-overrides-work

Reply:
Yah, either way. That's what I was thinking since my other projects tend to just use `.test.ts` instead of an explicit tests subdirectory.